api/src/paths/dwc/submission/{datasetId}/artifacts.ts

```diff
@@ -35,66 +36,71 @@ GET.apiDoc = {
       content: {
         'application/json': {
           schema: {
-            type: 'object',
-            required: ['artifacts'],
-            properties: {
-              artifacts: {
-                type: 'array',
-                items: {
+            type: 'array',
+            items: {
+              type: 'object',
```

Comment:
Any reason for removing the top level object?

It probably doesn't really matter at the moment, since we probably weren't sending more than the 1 artifacts field anyway, but it does mean we have no way to send additional top level data to this endpoint (if needed), without making breaking changes to add the object layer back.

Reply:
My consideration was the endpoint is designed for list of artifacts. Adding supplementary data inside the object seemed like it made more sense. Where in SIMS we actually contained that supplementaryData in that higher order object. IE:
`{ artifact: [], supplementaryData: [] }`
With the supplementaryData inside the artifact object now, I didn't see much consideration in having the artifact object. As any extra information per artifact will be inside, and any more global information would probably be housed at a different endpoint serving a higher order standard.